core/src/main/java/org/aerogear/mobile/core/metrics/MetricsService.java

```diff
@@ -79,6 +77,8 @@ public void sendAppAndDeviceMetrics(final Callback callback) {
     /**
      * Send metrics
      *
+     * @apiNote Not intended by direct usage outside of the SDK
```

Comment:
Another approach would be to avoid the internal API changes and just sprinkle these comments around.

Reply:
I really like this! IMHO that is all we need to do here.